portal/eproms/templates/eproms/base.html

```diff
@@ -59,7 +59,27 @@
             {%- endblock -%}
           </div> {# <!-- <div id="mainDiv"> --> #}
         </div> {# <!-- <div id="mainHolder"> --> #}
-
+        <div class="modal fade" tabindex="-1" role="dialog" id="urlAuthenticatedModal">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-label="{{ _('Close') }}"><span aria-hidden="true">&times;</span></button>
+                    <h2 class="modal-title">{{ _("Limited Access") }}</h2>
+                </div>
+                <div class="modal-body">
+                    <p>{{_("If you want to access that, you'll need to log in with your username and password.")}}</p>
```

Comment:
Noting that per slack, Mo will likely edit (that text was my rather offhand suggestion)